test/runtests.jl

```diff
@@ -44,12 +44,13 @@ include("models/normal.jl")
 include("models/normallognormal.jl")
 
 if TEST_GROUP == "All" || TEST_GROUP == "Interface"
-    # Interface tests that do not involve testing on Enzyme
+    #Interface tests that do not involve testing on Enzyme
```

Comment:
```suggestion
    # Interface tests that do not involve testing on Enzyme
```
is this intentional?

Reply:
No, seems like typo